UndertaleModLib/UndertaleChunks.cs

```diff
@@ -900,6 +900,19 @@ public class UndertaleChunkCODE : UndertaleListChunk<UndertaleCode>
     {
         public override string Name => "CODE";
 
+        private void CheckForGM2023_8(UndertaleReader reader)
+        {
+            if (!reader.undertaleData.IsVersionAtLeast(2, 3) || reader.undertaleData.IsVersionAtLeast(2023, 8))
```

Comment:
The minimum version check should be 2023.2 rather than 2.3 because of the PSEM chunk.